test/Remittance.js

```diff
@@ -0,0 +1,339 @@
+'use strict';
+
+const Remittance = artifacts.require('Remittance');
+const truffleAssert = require('truffle-assertions');
+
+const {toWei, fromWei } = web3.utils;
```

Comment:
`const { toWei, fromWei, BN, utf8ToHex } = web3.utils;`

Reply:
Not agreeing for the BN utility. Indeed I have some bug when calling BN functions. That's why I used the package big-number